sphinxcontrib/confluencebuilder/__init__.py

```diff
@@ -70,6 +70,12 @@ def assetsDocTreeResolvedHook(app, doctree, docname):
     app.add_config_value('confluence_publish_postfix', None, False)
     """Prefix to apply to published pages."""
     app.add_config_value('confluence_publish_prefix', None, False)
+    """Add section numbers if doctree has :numbered: option"""
+    app.add_config_value('confluence_add_secnumbers', False, False)
+    """Add section numbers to page titles if doctree has :numbered: option"""
+    app.add_config_value('confluence_add_pagesecnumbers', False, False)
```

Comment:
I actually wonder if this is best disabled by default. This extension, IMHO, should first and foremost should follow the practices promoted by the core Sphinx implementation and restructuredText specification (when applicable). If a documentation set using `toctree` with the `numbered` option by default generates section identifiers in, let us say, the HTML builder, it would think it would be best for this builder to do the same. While I understand it is not ideal change expected output through various revisions, I do not think the previous limitation of this extension (of not supporting `numbered` should require users properly using the option to also enable the feature as well. If in fact, by default, section numbers are disabled on other builders, this comment can be disregarded.

Reply:
I am of the same opinion, but opted for backward compatibility to increase the chance of this PR being accepted. I changed it to default on.